### DISCUSSION TITLE
Codespaces for Individual Pro Plan, how or when?

### DISCUSSION THREAD

**Discussion by redflitzi** [community/community#31222] [Codespaces]
Hi,
I would like to start with Codespaces as an individual developer.  But Codespaces is not available for me, probably because it is in Beta and I am not in a beta plan.
Unfortunately I also cannot find a way to apply for Codespaces Beta.

In May 2021 it was said that there will soon be communicated an upcoming Codepages plan for individuals. But nothing happened (or I missed it).
So how can Individuals get access to Codespaces? Is there a known way to still apply for the beta program? Or is here someone who knows when or whether Codespaces will be available for Individuals?
Thank you very much!

**Comment by jfullstackdev** [comment]
I joined the waitlist as I'm using a free account, but it was actually for several months before I have the free version

**Comment by redflitzi** [comment]
Thank you, but unfortunately the signup page for the waitlist has gone, says 404.
I am also currently using a free account and so would be happy with free Codespaces access, but if it will only be available to Pro accounts, this will also be ok for me.
